iocBoot/templates/Makefile.base

```diff
@@ -23,10 +24,25 @@ TEMPLATE_PATH ?= $(IOC_TOP)/iocBoot/templates
 STCMD_TEMPLATE ?= st.cmd.template
 IOC_NAME := $(shell basename "$(IOC_INSTANCE_PATH)")
 PRODUCTION_IOC ?= 0
-PYTMC_MIN_VERSION ?= 2.7.7
+PYTMC_MIN_VERSION ?= 2.13.0  # <-- TODO update before merging
```

Comment:
Calling out TODO in an isolated comment as a reminder for high visibility

Reply:
pytmc released and version has been bumped